packages/grid/x-data-grid-pro/src/tests/detailPanel.DataGridPro.test.tsx

```diff
@@ -39,6 +40,22 @@ describe('<DataGridPro /> - Detail panel', () => {
     );
   }
 
+  function PaginationTestCase({
```

Comment:
DOn't sure to get why introducing a `PaginationTestCase` in test file for detailPanel